### PR TITLE
Support Py3.9 + Django3.2, Drop Py3.5

### PR DESCRIPTION
**Pull request by alexandredufour**
Add support for Django 3.2.X
Add test matrix for Django 3.2.X

Add support for Python 3.9.X
Add test matrix for Python 3.9.X

Drop support for Python 3.5.X

Increment version => 1.8.10

Travis => https://travis-ci.org/github/rhblind/drf-haystack/builds/766795914

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -5,36 +5,41 @@ envlist =
...
     py36-django3.1-es2.x,
+    py36-django3.2-es1.x,
+    py36-django3.2-es2.x,
 
     py37-django2.2-es1.x,
     py37-django2.2-es2.x,
     py37-django3.0-es1.x,
     py37-django3.0-es2.x,
-    py36-django3.1-es1.x,
-    py36-django3.1-es2.x,
```
Comment (alexandredufour) [inline tox.ini:23]: These two never tested with py37

File: tox.ini [hunk 2]
```diff
@@ -5,36 +5,41 @@ envlist =
...
-
-    py38-django2.2-es1.x,
-    py38-django2.2-es2.x,
-    py38-django3.0-es1.x,
-    py38-django3.0-es2.x,
-    py36-django3.1-es1.x,
-    py36-django3.1-es2.x,
```
Comment (alexandredufour) [inline tox.ini:30]: The two never tested with py38

File: tox.ini [hunk 3]
```diff
@@ -5,36 +5,41 @@ envlist =
...
+    py38-django3.2-es2.x,
+
+    py39-django2.2-es1.x,
+    py39-django2.2-es2.x,
+    py39-django3.0-es1.x,
+    py39-django3.0-es2.x,
+    py39-django3.1-es1.x,
+    py39-django3.1-es2.x,
+    py39-django3.2-es1.x,
+    py39-django3.2-es2.x,
```
Comment (alexandredufour) [inline tox.ini:42]: Add Py39

**Comment by alexandredufour** [comment]
@rhblind 
Thanks for the quick merge.
Do you think it would be possible to create the `1.8.10` release in pypi ?

**Comment by rhblind** [comment]
Beautiful, thanks!
Yes, I'll try to get it done within a few days :)

**Comment by jgadelange** [comment]
Any update on the ETA? (preferably also a release with #160 👼 )
Or should I for now add a git install line to my requirements.txt?

**Comment by rhblind** [comment]
Hey,
I'm sorry, I got caught up in work. I'll put it in my calendar and make a release before the weekend!

**Comment by rhblind** [comment]
Hello,
I really appreciate the work you guys do to help me keeping this project somewhat still relevant, and I'm sorry for being slow to merge and respond to your PRs. But anyways, thank you very much ♥️

Just pushed the new version to PyPi now!

And, by the way, if anybody wants to help maintaining the library, please shout out!

**Pull request merged**